### PR TITLE
v1.9 backports 2021-01-18

### PR DESCRIPTION
**Pull request by aditighag**
* #14524 -- jenkinsfile: Allow enabling host firewall in k8s-all CI (@pchaigno)
 * #14624 -- docs: AKS - remove azure0 from CNI chaining config (@ti-mo)
 * #14623 -- pkg/kvstore: fix panic when closing etcd connections on error (@aanm)

Once this PR is merged, you can update the PR labels via:
```upstream-prs
$ for pr in 14524 14624 14623; do contrib/backporting/set-labels.py $pr done 1.9; done
```

### CODE REVIEW

**Review by pchaigno — CHANGES_REQUESTED** [review]
@aditighag I think something went wrong here and only one of the three aforementioned PRs was backported.

**Comment by aditighag** [comment]
> @aditighag I think something went wrong here and only one of the three aforementioned PRs was backported.

Yes, I'm aware, and fixing it.

**Comment by aditighag** [comment]
test-backport-1.9

File: jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile [hunk 1]
```diff
@@ -39,7 +39,8 @@ pipeline {
         SERVER_BOX = "cilium/ubuntu"
         FAILFAST=setIfLabel("ci/fail-fast", "true", "false")
         CNI_INTEGRATION=setIfLabel("integration/cni-flannel", "FLANNEL", "")
-        GINKGO_TIMEOUT="98m"
+        HOST_FIREWALL=setIfLabel("ci/host-firewall", "1", "0")
+        GINKGO_TIMEOUT="118m"
```
Comment (pchaigno) [inline jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile]: I think this line was committed by mistake (probably due to merge conflict).

Comment (aditighag) [reply jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile]: Yes, I had to resolve merge conflicts. So should I revert this to `GINKGO_TIMEOUT="98m"`? Looks like the timeout was changed in some other PR that's not backported to 1.9.

Comment (pchaigno) [reply jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile]: We can either label and backport that other PR or revert to `GINKGO_TIMEOUT="98m"`. I don't think we've hit the timeout issue on v1.9, but maybe it's just a matter of time.

Comment (aditighag) [reply jenkinsfiles/ginkgo-kubernetes-all.Jenkinsfile]: I've reverted the value. /cc @nebril

**Comment by aditighag** [comment]
test-backport-1.9

**Review by pchaigno — APPROVED** [review]

**Review by ti-mo — APPROVED** [review]

**Review by aanm — APPROVED** [review]
Looks good for my commit, thank you!

**Comment by aanm** [comment]
test-gke

**Comment by aditighag** [comment]
test-missed-k8s

**Comment by aditighag** [comment]
Infrastructure issue? Re-test test-k8s.

>17:17:45      k8s2-1.14: Setting up kubelet (1.14.10-00) ...
17:17:45      k8s2-1.14: Created symlink /etc/systemd/system/multi-user.target.wants/kubelet.service -> /lib/systemd/system/kubelet.service.
17:17:45      k8s2-1.14: Setting up kubectl (1.14.10-00) ...
17:17:45      k8s2-1.14: Setting up kubeadm (1.14.10-00) ...
17:18:07  Cancelling nested steps due to timeout
17:18:07  Sending interrupt signal to process
17:18:20  script returned exit code 255

**Comment by aditighag** [comment]
test-missed-k8s

**Pull request merged**